packages/blockchain/src/consensus/clique.ts

```diff
@@ -233,17 +244,24 @@ export class CliqueConsensus implements Consensus {
       }
     }
 
+    this._cliqueLatestSignerStates.sort((a, b) => (a[0] > b[0] ? 1 : -1))
```

Comment:
Is it likely that the clique signer states will get out of order and require us to potentially sort them twice in a single call to this update method?

Reply:
I added this just to be sure, but looking at the code, it trims the `cliqueLatestSignerStates` after the first sort (so they are already ordered by block number) and thus removes latest signer states if they are below a certain threshold number (so the order should still be fine). I think we can remove.